scripts/aggregate-translations/write-aggregated-translations.js

```diff
@@ -12,7 +12,23 @@ const writeAggregatedTranslations = (aggregatedMessages, locales, fileSystem, ou
       }
 
       const translationFilePath = path.resolve(outputDir, `${locale}.js`);
-      fileSystem.writeFileSync(translationFilePath, generateTranslationFile(locale, aggregatedMessages[locale]));
+      const messages = aggregatedMessages[locale];
+      const localeRegionSplit = locale.split('-');
+      let mergedMessages = messages;
+
+      // Check if this is a regional locale
+      if (localeRegionSplit.length > 1) {
+        const localeMessages = aggregatedMessages[localeRegionSplit[0]];
+        if (localeMessages) {
```

Comment:
If this searches `en-AU`, then `en` will this provided `en` fallback or do we need to sort the locale list before aggregating to ensure the base locale messages have been found?

Reply:
All the translation aggregating happens before this function is called, so there shouldn't be an issue.